.pipelines/build-model-container-image.Jenkinsfile

```diff
@@ -40,19 +41,17 @@ pipeline {
             steps {
                 echo "Hello docker image build ${env.BUILD_ID}"
                 checkout scm
-                //checkout([$class: 'GitSCM', branches: [[name: '*/ml_model_uc81']],
-                //    userRemoteConfigs: [[url: 'https://github.com/Merlion-Crew/MLOpsPython.git/']]])
 
                 sh '''
-                    conda env create --file ./diabetes_regression/ci_dependencies.yml --force 
+                    conda env create --name $DEFAULT_CONDA_ENV_NAME --file ./diabetes_regression/ci_dependencies.yml --force 
                 '''
                 
                 withCredentials([azureServicePrincipal("${AZURE_SP}")]) {
                     sh '''#!/bin/bash -ex
                         az login --service-principal -u $AZURE_CLIENT_ID -p $AZURE_CLIENT_SECRET -t $AZURE_TENANT_ID
                         az account set -s $AZURE_SUBSCRIPTION_ID
                         export SUBSCRIPTION_ID=$AZURE_SUBSCRIPTION_ID
-                        source /home/azureuser/anaconda3/bin/activate mlopspython_ci
+                        source /home/azureuser/anaconda3/bin/activate $DEFAULT_CONDA_ENV_NAME
```

Comment:
Can we do a `conda activate <ENV_NAME>` here? Not so sure if the in the future that path for conda executables would change though. 😄

Reply:
can't use 'conda activate', I updated it with ANACONDA PATH environment in Jenkins nodes.